### PR TITLE
e2e: Add kube-hunter configs and script

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: scripts/kube-hunter.sh [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+while [ "$(kubectl get "${pod_name}" -o jsonpath='{.status.phase}')" != "Succeeded" ]; do
+  echo "Waiting until the 'kube-hunter' job is complete..."
+  sleep 1
+done
+
+echo "--------------------------------------------------"
+echo "kube-hunter output:"
+echo "--------------------------------------------------"
+kubectl logs "${pod_name}"
```
Comment (invidian) [inline scripts/kube-hunter.sh:58]: Should we somehow check the status of the kube-hunter here and exit with error if it found something?

File: scripts/kube-hunter-manifests/config.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (invidian) [inline scripts/kube-hunter-manifests/config.yaml:1]: We could consider using helm chart in the future: https://github.com/helm/charts/tree/master/stable/kube-hunter.

**Review by invidian — APPROVED** [review]
Just some questions, otherwise LGTM

File: scripts/kube-hunter-manifests/config.yaml (hunk 2 above)
Comment (surajssd) [reply scripts/kube-hunter-manifests/config.yaml:1]: Yep, but it lacks psp related settings.

File: scripts/kube-hunter.sh (hunk 1 above)
Comment (surajssd) [reply scripts/kube-hunter.sh:58]: Currently the part of the story/work is to just post the output. And as a followup work is to create the error reporting mechanism.

**Comment by surajssd** [comment]
The original commit failed with following error:

```bash
+ pushd ../..
/tmp/build/8c0986c9/lokomotive-kubernetes /tmp/build/8c0986c9/lokomotive-kubernetes/ci/aws
+ RUN_FROM_CI='"true"'
+ make run-e2e-tests
KUBECONFIG="/root/assets/auth/kubeconfig" ./scripts/kube-hunter.sh
W1107 14:47:25.279377     467 loader.go:223] Config not found: /root/assets/auth/kubeconfig
unable to recognize "/tmp/build/8c0986c9/lokomotive-kubernetes/scripts/kube-hunter-manifests/config.yaml": Get http://localhost:8080/api?timeout=32s: dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "/tmp/build/8c0986c9/lokomotive-kubernetes/scripts/kube-hunter-manifests/config.yaml": Get http://localhost:8080/api?timeout=32s: dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "/tmp/build/8c0986c9/lokomotive-kubernetes/scripts/kube-hunter-manifests/config.yaml": Get http://localhost:8080/api?timeout=32s: dial tcp 127.0.0.1:8080: connect: connection refused
make: *** [Makefile:11: kube-hunter] Error 1
+ RET=2
+ popd
/tmp/build/8c0986c9/lokomotive-kubernetes/ci/aws
```

**Comment by surajssd** [comment]
Got a weird error this time:

```
--------------------------------------------------
unable to recognize "/tmp/build/8c0986c9/lokomotive-kubernetes/scripts/kube-hunter-manifests/config.yaml": Get https://loko-ci-1573139279.test.lokomotive-k8s.net:6443/api?timeout=32s: dial tcp 3.124.161.255:6443: connect: connection refused
unable to recognize "/tmp/build/8c0986c9/lokomotive-kubernetes/scripts/kube-hunter-manifests/config.yaml": Get https://loko-ci-1573139279.test.lokomotive-k8s.net:6443/api?timeout=32s: dial tcp 3.124.87.148:6443: connect: connection refused
unable to recognize "/tmp/build/8c0986c9/lokomotive-kubernetes/scripts/kube-hunter-manifests/config.yaml": Get https://loko-ci-1573139279.test.lokomotive-k8s.net:6443/api?timeout=32s: dial tcp 3.124.87.148:6443: connect: connection refused
make: *** [Makefile:11: kube-hunter] Error 1
+ RET=2
+ popd
```

**Comment by surajssd** [comment]
Packet tests are failing because of unavailability of the machines.

**Comment by surajssd** [comment]
The output of the CI run on AWS:

```bash
Apply complete! Resources: 105 added, 0 changed, 0 destroyed.
+ '[' 0 = 0 ']'
+ pushd ../..
/tmp/build/8c0986c9/lokomotive-kubernetes /tmp/build/8c0986c9/lokomotive-kubernetes/ci/aws
+ RUN_FROM_CI='"true"'
+ make run-e2e-tests
KUBECONFIG="/root/assets/auth/kubeconfig" ./scripts/kube-hunter.sh
The connection to the server loko-ci-1573198900.test.lokomotive-k8s.net:6443 was refused - did you specify the right host or port?
Waiting for the cluster to be responsive...
The connection to the server loko-ci-1573198900.test.lokomotive-k8s.net:6443 was refused - did you specify the right host or port?
Waiting for the cluster to be responsive...
The connection to the server loko-ci-1573198900.test.lokomotive-k8s.net:6443 was refused - did you specify the right host or port?
Waiting for the cluster to be responsive...
The connection to the server loko-ci-1573198900.test.lokomotive-k8s.net:6443 was refused - did you specify the right host or port?
Waiting for the cluster to be responsive...
The connection to the server loko-ci-1573198900.test.lokomotive-k8s.net:6443 was refused - did you specify the right host or port?
Waiting for the cluster to be responsive...
The connection to the server loko-ci-1573198900.test.lokomotive-k8s.net:6443 was refused - did you specify the right host or port?
Waiting for the cluster to be responsive...
NAME             STATUS   ROLES    AGE   VERSION
ip-10-0-0-153    Ready    <none>   61s   v1.16.2
ip-10-0-23-197   Ready    <none>   60s   v1.16.2
ip-10-0-37-216   Ready    <none>   59s   v1.16.2
serviceaccount/kube-hunter created
clusterrolebinding.rbac.authorization.k8s.io/privileged-psp-kube-hunter created
job.batch/kube-hunter created
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
The pod has not started yet...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
Waiting until the 'kube-hunter' job is complete...
--------------------------------------------------
kube-hunter output:
--------------------------------------------------
~ Started
~ Discovering Open Kubernetes Services...
```

**Review by invidian — APPROVED** [review]

**Pull request merged**